src/Host/Client/Impl/Definitions/IRBlob.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.R.Host.Client {
+    public interface IRBlob {
```

Comment:
If this implements `Dispose` and destroys the blob inside that, it can be conveniently used with `using`.